.circleci/config.yml

```diff
@@ -5,11 +5,13 @@ defaults: &defaults
     - run:
         name: Install skt using pip
         command: pip install .
+    # Disable R0801 in pylint that checks for duplicate content in multiple
```

Comment:
I think this would've been clearer, if it said something like "Run pylint with R0801 disabled..." or "Running with R0801 disabled in pylint...". As the main thing here is that we actually run pylint, and not just disabling stuff.

Although a little confusing, this can go in as is.